js/interactive-guides/step-content.js

```diff
@@ -109,8 +166,36 @@ var stepContent = (function() {
     var instructionNumber = contentManager.getCurrentInstructionIndex(stepName);    
     if(currentInstruction){
       currentInstruction = __addInstructionTag(stepName, currentInstruction, instructionNumber);
```

Comment:
This block of codes (lines 168 thru 176) is the same as that from lines 96 thru 104. Could we create a common method?

Reply:
One of them is the instruction block (green block) and the other is looking for instruction contents (the different instructions inside of the green block).